src/scss/global.scss

```diff
@@ -7,6 +7,13 @@
 @import "node_modules/bootswatch/dist/darkly/bootswatch";
 @import "./constants.scss";
 
+html,
+body,
+#root {
+  height: 100%;
+  width: 100%;
+}
+
```

Comment:
Why adding this? I don't think we need to do it, and I think it may even be an antipattern (at least so it seemed in some of the things I was reading yesterday)

Reply:
Try removing this. It breaks the scrolling capabilities.

Reply:
Just pulled down your latest changes + toggled them off. Scrolling seems to work fine in the auditorium/etc; and looking at the calculated height/width of each element in the 'chain' going down seems like they are correct?

Where are you seeing it not work/in what situations?

Reply:
Using this size for all of the below cases:
![image](https://user-images.githubusercontent.com/753891/110092090-51eade00-7ded-11eb-85e7-fa3d14314b3f.png)

---

Using these styles:
![image](https://user-images.githubusercontent.com/753891/110092063-4992a300-7ded-11eb-85c1-942942d63ccb.png)

The computed size is:
![image](https://user-images.githubusercontent.com/753891/110092134-5ca57300-7ded-11eb-88a7-3973382bdd8f.png)

---

Using these styles:
![image](https://user-images.githubusercontent.com/753891/110092193-6d55e900-7ded-11eb-95fb-ede2ceec925f.png)

The computed size is:
![image](https://user-images.githubusercontent.com/753891/110092222-7646ba80-7ded-11eb-906f-1f69dc9a43db.png)

---

Unsetting these styles completely:
![image](https://user-images.githubusercontent.com/753891/110092267-85c60380-7ded-11eb-907c-c623ecbe4688.png)

The computed size is:
![image](https://user-images.githubusercontent.com/753891/110092293-8d85a800-7ded-11eb-9cbe-87a42108dfab.png)

Reply:
It doesn't allow to scroll to the left side

Uploading Screen Recording 2021-03-05 at 12.01.11.mov…

Reply:
FYI: you can attach videos directly to GitHub issues/comments/etc now, just drag/drop/upload it.

Reply:
https://user-images.githubusercontent.com/63194656/110104473-df0f4200-7daf-11eb-96fc-20bc7c191da5.mov

Reply:
Yup.. I see the issue. Had a quick play/look around and not sure why tbh.

Also had a quick google and couldn't find any sources backing up my "it's an antipattern" thoughts from whatever I was reading yesterday either.

So based on that, I say let's keep it! 👍🏻

Reply:
It lacks some perspective ^, but basically, I can scroll to the right side(to the end), but can't do it to the left side

Reply:
_nods_ yeah, I saw that. And while it frustrates me that I can't explain why that is. Your change fixes it. So :shipit: